scripting/include/get5.inc

```diff
@@ -5,6 +5,7 @@
 
 #include <json>  // github.com/clugg/sm-json
 #include <cstrike>
+#include <SteamWorks>
```

Comment:
With this include, we should probably remove the include in `get5.sp`, as that file already includes `get5.inc`. Not sure about this though.